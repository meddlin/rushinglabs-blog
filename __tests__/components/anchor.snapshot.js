import { render } from '@testing-library/react';
import Anchor from '../../components/content-helpers/anchor';

it('renders Anchor unchanged', () => {
    const { container } = render(
        <Anchor href="https://rushinglabs.com">this is a test</Anchor>
    );

    expect(container).toMatchSnapshot()
});