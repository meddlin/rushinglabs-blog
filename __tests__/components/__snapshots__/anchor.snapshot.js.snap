// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Anchor unchanged 1`] = `
<div>
  <a
    class=" underline underline-offset-1 decoration-solid decoration-slate-400 decoration-1 hover:decoration-indigo-500 hover:decoration-2"
    href="https://rushinglabs.com"
  >
    this is a test
  </a>
</div>
`;
